cli/src/commands/status.rs

```diff
@@ -72,7 +72,14 @@ pub(crate) fn cmd_status(
                 formatter.labeled("conflict"),
                 "There are unresolved conflicts at these paths:"
             )?;
-            resolve::print_conflicted_paths(&conflicts, formatter, &workspace_command)?
+            resolve::print_conflicted_paths(&conflicts, formatter, &workspace_command)?;
+
+            print_conflict_resolution_hint(
+                formatter,
+                "To resolve the conflicts, either work on the current change or create a new one \
+                 to squash later.",
+                &[],
```

Comment:
I think we would ideally want to tell the user to `jj new` to the first commit with conflicts, but I think this is already better than before, so we can improve it in a separate PR.